### PR TITLE
mrc-1982 fetch docs endpoints and render docs

### PR DESCRIPTION
**Pull request by hillalex**
This touches quite a lot of files but is very boiler-platey - let me know if you'd prefer I split it up.

Docs are returned from mintr in 2 ways - as properties on the dynamic form settings, where docs apply to form sections, and through 2 independent endpoints for intepreting figures/results: https://github.com/mrc-ide/mintr/pull/28

This PR adds:
* 2 new GET endpoints `impact/docs`, `cost/docs`
* corresponding Vuex store props, actions and mutations
* a new component `collapsibleDocs` for rendering the docs on each of the impact and cost tabs
* a few css rules to render the docs coming back from mintr with the updated settings

### CODE REVIEW

**Review by EmmaLRussell — COMMENTED** [review]
Just starting to look at this - looks like front end test failures following the package update?

File: src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt [hunk 1]
```diff
@@ -31,4 +31,10 @@ class ImpactTests: EndpointTests() {
         val responseEntity = testRestTemplate.getForEntity<String>("/impact/table/config")
         assertSuccess(responseEntity, "TableDefinition")
     }
+
+    @Test
+    fun `can get impact docs`() {
```
Comment (EmmaLRussell) [inline src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt:36]: missing test for cost docs here?

File: src/config/mintr_version [hunk 2]
```diff
@@ -1 +1 @@
-master
+mrc-1982
```
Comment (EmmaLRussell) [inline src/config/mintr_version:1]: I think this can reverted to master now

File: src/app/static/src/app/components/projectListPage.vue [hunk 3]
```diff
@@ -1,22 +1,38 @@
 <template>
-     <div class="container mt-5">
+    <div class="container mt-5">
         <div class="row">
             <div class="col-lg-6 offset-lg-3 col-10 offset-1">
+                <p>
+                    This tool is designed to help National Malaria Control Programs explore the most cost effective
```
Comment (EmmaLRussell) [inline src/app/static/src/app/components/projectListPage.vue:6]: I wonder if this should be collapsible too, and collapsed by default if the user has at least one project i.e. we can assume they've already seen it. It feels a little intrusive having this here when you want to be editing projects..

**Review by EmmaLRussell — COMMENTED** [review]
Looks good!

Do we need a ticket to add the mentioned 'red dashed line' to the cost effecitveness plots?

'how to interpret these figures' should probably be captialised to be consistent with 'How to use these settings'

In the 'Price of Interventions' docs, looks like there's a missing para break before 'Price of spray product per person (annual $USD)' - that'll be a mintr change though.

File: src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt (hunk 1 above)
Comment (hillalex) [reply src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt:36]: that should be in `CostTests` class :)  https://github.com/mrc-ide/mint/pull/66/files#diff-b145b19b1864f9e5b9a7ee9bbd72aa6ca12e19750919ae2df3c83daf79fb14abR28

**Comment by hillalex** [comment]
> Looks good!
> 
> Do we need a ticket to add the mentioned 'red dashed line' to the cost effecitveness plots?
> 
> 'how to interpret these figures' should probably be captialised to be consistent with 'How to use these settings'
> 
> In the 'Price of Interventions' docs, looks like there's a missing para break before 'Price of spray product per person (annual $USD)' - that'll be a mintr change though.

ha, yes just noticed the bit about the "red dashed line"! will change casing and look into package update test failures

**Comment by hillalex** [comment]
Ok, have reverted the jquery update which `npm audit fix` added and created a separate ticket for that - https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1997

File: src/app/static/src/app/components/projectListPage.vue (hunk 3 above)
Comment (hillalex) [reply src/app/static/src/app/components/projectListPage.vue:6]: I dunno, we have a lot of collapsible content now, feels like this is ok as contextual info about projects/regions to display all the time? I'm minded to leave as is unless anything is raised by others in user testing?

File: src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt (hunk 1 above)
Comment (EmmaLRussell) [reply src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt:36]: :facepalm:

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**